recipes/quaternionic/meta.yaml

```diff
@@ -1,27 +1,27 @@
 {% set owner = "moble" %}
 {% set name = "quaternionic" %}
-{% set version = "0.1.0" %}
+{% set version = "0.1.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/{{ owner }}/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: c83bd808174621c38b96a103b14a772733cb70e874800170ae15bb132b439019
+  sha256: cbbd78af8e826af291c7dad67436c8fa65f0366e10b5619f52581df20d402cf8
 
 build:
+  noarch: python
   number: 0
   script: "{{ PYTHON }} -m pip install . -vv"
-  skip: True  # [py<36]
 
 requirements:
   host:
     - python
-    - pip >=19.0
+    - pip >=20.0.2
     - poetry
   run:
-    - python
+    - python >=3.6
```

Comment:
@scopatz Is this right?

Reply:
both pythons should have the same pin, but other than that it looks right!